lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -34,7 +35,7 @@ class UnusedBytes extends Audit {
    * @return {string}
    */
   static bytesToKbString(bytes) {
-    return Math.round(bytes / KB_IN_BYTES).toLocaleString() + ' KB';
+    return `${Util.formateBytesToKB(bytes)} KB`;
```

Comment:
you inherited this typo but do you want to update it to `formatBytesToKB` while you're in here? :)